modules/connatixBidAdapter.js

```diff
@@ -217,7 +234,42 @@ export const spec = {
       type: 'iframe',
       url
     }];
-  }
+  },
+
+  /**
+   * Register bidder specific code, which will execute if the server response time is greater than auction timeout
+   */
+  onTimeout: (timeoutData) => {
+    const connatixBidRequestTimeout = timeoutData.find(bidderRequest => bidderRequest.bidder === BIDDER_CODE);
```

Comment:
Couple things here:

If a publisher uses an alias for `connatix` then this will not find it - you can do this by keep a list of alias names that you see come in through `buildRequests` - or you can import the aliasRegistry I think.

Although - I do think that this method will only receive an array of bids that went through connatix adapter - and therefore you might not even need to find / filter etc.

Reply:
@robertrmartinez Thanks for the heads-up! We’ve incorporated aliasRegistry to manage aliases, streamlining the process and eliminating the need to manually track them in buildRequests. This should ensure Connatix aliases are automatically handled. Could you please take another look at the PR when you have a moment? Thanks!